manage_breast_screening/participants/jinja2/components/participant-details/screening_details.jinja

```diff
@@ -0,0 +1,63 @@
+{% from 'card/macro.jinja' import card %}
+{% from 'summary-list/macro.jinja' import summaryList %}
+
+{% macro screening_details(presented_mammograms, screening_protocol) %}
+  {% set mammograms_by_date_added = presented_mammograms.last_known_mammograms %}
+  {% set last_mammogram_html %}
+    <div data-testid="mammograms">
+    {% if mammograms_by_date_added %}
+      {% for mammogram in mammograms_by_date_added %}
+        <p>
+        <span class="nhsuk-u-font-weight-bold">Added {{ mammogram.date_added }}</span>
+        <br>{% if mammogram.date.is_exact %}{{mammogram.date.absolute}} <span class="app-text-grey">({{mammogram.date.relative}})</span>{% else %}{{ mammogram.date.value }}{% endif %}
+        <br>{{ mammogram.location }}
+
+        {% if mammogram.different_name %}
+          <br>Previous name: {{ mammogram.different_name }}
+        {% endif %}
+
+        {% if mammogram.additional_information %}
+          <br>Additional information: {{ mammogram.additional_information | nl2br }}
```

Comment:
Just me liking nicely indented HTML

If you wanted the new lines to match the (10 space) indent level of the first line:

```suggestion
          <br>Additional information: {{ mammogram.additional_information | nl2br | indent(10) }}
```